app/templates/components/filter-by.hbs

```diff
@@ -1,10 +1,61 @@
-{{#if isShowingBody}}
-  <div class="filter-table">
-    <ul>
-      <li>Teams</li>
+{{!-- Filter By --}}
+
+<div class="filter">
+  <BsDropdown as |dd|>
+    <dd.toggle class="filter-by"><span>Filter by</span><p
+        class="emp"
+        {{action "filterby"}}
+      >{{if currentTeam currentTeam "All Employees"}}</p>
+      <img
+        src="/assets/images/accordion-open.svg"
+        alt="img"
+        class="open {{if clicked 'turn'}}"
```

Comment:
Instead of having one more image and class to show the up and down arrow, You can use the bs-dropdown arrow itself right..? You can change the styles based on the mock.

Reply:
waiting for your solution